conftest.py

```diff
@@ -30,6 +30,15 @@ def __str__(self):
         return "{} (port {})".format(self.message, self.port)
 
 
+def pytest_configure(config):
+    if store.current_appliance.version < '5.6':
```

Comment:
can we use current_appliance from the appliance module in order to avoid using store?